packages/marketing-modal/src/MarketingModal.tsx

```diff
@@ -15,6 +16,14 @@ const Mode = {
 
 type Mode = typeof Mode[keyof typeof Mode];
 
+export const BlobPosition = {
+  TopLeft: 'top left',
+  TopRight: 'top right',
+  BottomRight: 'bottom right',
+} as const;
```

Comment:
No BottomLeft?

Reply:
No, there were only 3 options in the design.

Reply:
Hmm interesting. I'd double check and make sure that's intentional

Reply:
Yup, Sooa confirmed that there are only 3.